src/components/Modals/ImportWithdrawalModal/ImportWithdrawalModal.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { ComponentProps, useEffect, useState } from 'react'
+import { Close, Button, Field } from 'decentraland-ui'
+import Modal from 'decentraland-dapps/dist/containers/Modal'
+import { T, t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { getTransactionOrigin } from 'decentraland-dapps/dist/modules/transaction/utils'
+import { getChainIdByNetwork } from 'decentraland-dapps/dist/lib/eth'
+import { Network } from '@dcl/schemas'
+import { Props } from './ImportWithdrawalModal.types'
+
+import './ImportWithdrawalModal.css'
+
+const ImportWithdrawalModal = ({
+  withdrawals,
+  address,
+  name,
+  isLoading,
+  error,
+  onClose,
+  onImport,
+  onClearError,
+}: Props) => {
+  const [tx, setTx] = useState('')
+  const [txError, setTxError] = useState<string | undefined>()
+
+  useEffect(() => {
+    onClearError()
+  }, [onClearError])
+
+  const polygonscanHref = `${getTransactionOrigin(
+    getChainIdByNetwork(Network.MATIC)
+  )}/address/${address}#tokentxns`
```

Comment:
Is there something like this in @decentraland/dapps ?

Reply:
Couldn't find anything to form the scan url for an address.

Reply:
BTW, I'm opening the token transactions of the scan website because that's were the pertinent transactions are

Reply:
You could do `getTransactionHref({ address }, getChainIdByNetwork(Network.MATIC)) + '#tokentxs'` if you want

Reply:
Better! Thanks